pages/mobile/base.py

```diff
@@ -24,6 +25,10 @@ def menu_items(self):
         return [self.MenuItem(self.testsetup, element)
                 for element in self.selenium.find_elements(*self._menu_items_locator)]
 
+    @property
+    def mobile_view(self):
+        return self.selenium.find_element(*self._mobile_header)
```

Comment:
This is going to return the element, which is likely not what you want. Perhaps

`return self.selenium.find_element(*self._mobile_header).is_displayed()`

would be better.